doc/prscheduler.md

```diff
@@ -109,12 +119,15 @@ The playbook will prompt you for variables:
 ```
 # ansible-playbook -i ansible/hosts/runners ansible/prepare_openclose_pr_tool.yml
 Provide an unique identifier for the PR: testing_master    # $identifier
-Branch name to open PR against it: master                  # $branch
+Flow name to identify the nightly template: ci or pki      # $flow
 Repo owner to create branches: freeipa-pr-ci               # $repo_owner
 GitHub token: 123qwerty                                    # $github_token
 Provide the path to a test definition file to use it to open new PRs. For example: ipatests/prci_definition/gating	# $prci_config, for instance ipatests/prci_definition/nightly_master.yaml
 Provide a ssh key that has push access to the git repo: ~/path/to/ssh_key #$git_sshkey
 Should the PR be open against the upstream repo?. Type no for opening it agaist your own freeipa repo (yes/no): no
+Provide PRCI test definitions folder E.g. ipatests/prci_definition # $prci_def_dir
+Provide Fedora version (int) for template creation. E.g. 29 # $fedora_ver
+Provide Vagrant atlas token for uploading boxes # $vagrant_atlas_token
 ```
 
 You can run the ansible playbook multiple times to schedule multiple PRs,
```

Comment:
Now running the playbook once is actually running the script multiple times, once for each nightly_jobs described in automation/nightly_pr/defaults/main.yaml, and it will schedule all the PRs (testing_master, test_f28 etc...).